src/Discord.Net.Core/Entities/Permissions/ChannelPermission.cs

```diff
@@ -1,40 +1,29 @@
 ﻿namespace Discord
 {
-    public enum ChannelPermission : byte
+    public enum ChannelPermission : ulong
     {
-        //General
-        CreateInstantInvite = 0,
-        //KickMembers = 1,
-        //BanMembers = 2,
-        //Administrator = 3,
-        ManageChannel = 4,
-        //ManageGuild = 5,
+        CREATE_INSTANT_INVITE =     0x00000001,
```

Comment:
Also, the comments categorizing each section of permissions would be nice to still have in source. And maybe put an `_` every two digits for readability.